### PR TITLE
[Unit Tests] Added tests for connectivity.py

### PR DESCRIPTION
**Pull request by bachnguyenTE**
**Update part of the unit test to include comprehensive testing of existing functions in `connectivity.py`**

_New module:_ `test_connectivity.py`
- **[ENH]** Unit test for `connectivity.py`; approximately 70% coverage, major functions included, explanations included in comments
- Some functions and tests are excluded due to a currently unknown problem with the connectivity scaling function (scaling function changing connectome connectivity)

~~_Updated module:_ 'connectivity.py'~~
- ~~**[ENH]** The `scale` function now ignores zero values (non-connections) in the connectivity matrix, allowing for negative value min-max scaling and implementing inhibitory connections~~
- ~~**[ENH]** Modified `get_nodes` to allow inputting Python list when the user wants to select a list of subnetworks instead of just one without having to call and concatenate several `get_nodes` outputs~~
- ~~**[FIX]** Quick fix to bool type, same as @anaston+~~

### CODE REVIEW

**Review by estefanysuarez — CHANGES_REQUESTED** [review]

File: conn2res/connectivity.py
[the remark this answers is not in the document]
Comment (bachnguyenTE) [reply conn2res/connectivity.py:1]: Sure! I saw that @anaston also addressed one of the bugs that I fixed in the file, so I'll review and separate this file into a different one. From here on I'll use a separate tag for unit tests.

File: .gitignore
[the remark this answers is not in the document]
Comment (bachnguyenTE) [reply .gitignore:1]: Yes, during testing I created a file for the test that slipped `gitignore`. I'll keep that in mind from now on. Thanks for pointing it out!

File: conn2res/tests/test_connectivity.py
[the remark this answers is not in the document]
Comment (bachnguyenTE) [reply conn2res/tests/test_connectivity.py:1]: Yes, this unit test and future tests I work on will not affect the functionalities of any part of the repo. Sometimes test files have to be created (mostly small buffer files), but I'll try looking at ways to remove them after testing. 

Commented lines are mostly test functions that I created but there are unresolved or unknown problems in the source code that render them temporarily unusable or not insightful. I put that in as a note for myself to come back or if there is anyone else who wants to work on it, though I'll also see if I can make a note/doc in the unit test folder itself to make things clearer. Thank you!

**Comment by bachnguyenTE** [comment]
@estefanysuarez, thank you for your comments! I've reverted the changes made to `connectivity.py` to do a different PR later, and I've also noted down some files and steps to remove redundant test files in the future automatically. I edited the PR accordingly so let me know if anything needs to be polished before merging.

File: conn2res/connectivity.py
[the remark this answers is not in the document]
Comment (estefanysuarez) [reply conn2res/connectivity.py:1]: Perfect, thanks!

**Review by estefanysuarez — APPROVED** [review]
I'll look at the details later, but this is great for now Bach. This will be super helpful. Thank you very much!

**Pull request merged**